test/e2e/rmi_test.go

```diff
@@ -275,4 +275,12 @@ RUN find $LOCAL
 		Expect(images.ExitCode()).To(Equal(0))
 		Expect(len(images.OutputToStringArray())).To(Equal(0))
 	})
+
+	// Don't rerun all tests; just assume that if we get that diagnostic,
+	// we're getting rmi
+	It("podman image rm is the same as rmi", func() {
+		session := podmanTest.Podman([]string{"image", "rm"})
+		Expect(session.ExitCode()).To(Equal(125))
```

Comment:
Ed you need to add this line before the first 'Expect()' line:

session.WaitWithDefaultTimeout()

Reply:
Fixed and force pushed. Thank you @TomSweeneyRedHat